index.js

```diff
@@ -6,30 +6,16 @@ var xtend = require("xtend")
 
 module.exports = createXHR
 createXHR.XMLHttpRequest = window.XMLHttpRequest || noop
-createXHR.XDomainRequest = "withCredentials" in (new createXHR.XMLHttpRequest()) ? createXHR.XMLHttpRequest : window.XDomainRequest
 createXHR.qsSerialize = null // Define this as a function to support the `qs` option
 
-forEachArray(["get", "put", "post", "patch", "head", "delete"], function(method) {
+"get,put,post,patch,head,delete".split(",").map(function(method) {
```

Comment:
> @djake Seeing you both default to array syntax, I think I'll undo this golfing

👍

> then the only reason to use this instead of .fetch would be to get an xhr that you can abort.

For the record, this is precisely why I use this lib. Promises provide a subset of functionality compared to regular XHR's, yet are simple to make from this callback-style API, so I'm surprised anyone chooses to use `fetch`!